### PR TITLE
Update puppet scripts for OB200

### PR DESCRIPTION
**Pull request by DoryFazna**
## Purpose
To Run 200 obiam, obam , obbi for UK,Berlin, AU.

## Goals
To run and test OB-2.0.0 products.

## Approach
Compared OB200 scripts and configs and updated the changes.
Tested with all specs.

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

**Review by DivyaPremanantha — CHANGES_REQUESTED** [review]
Please do the changes required for the readme file and @isharailanga please review that

**Comment by Dinushi** [comment]
WUM updates for UK MI Reporting siddhi files for UK 3.1.5 were also released. Shall we create a issue to track this requirement to add UK 3.1.5 siddhi-files also to puppet.

**Comment by isharailanga** [comment]
> Please do the changes required for the readme file and @isharailanga please review that

Hi @DivyaPremanantha @DoryFazna,

Ack. Please keep me posted.

**Comment by DoryFazna** [comment]
> WUM updates for UK MI Reporting siddhi files for UK 3.1.5 were also released

Created the issue. Thanks

**Comment by DoryFazna** [comment]
> Please do the changes required for the readme file and @isharailanga please review that

Updated. Thanks.

> > Please do the changes required for the readme file and @isharailanga please review that
> 
> Hi @DivyaPremanantha @DoryFazna,
> 
> Ack. Please keep me posted.

Updated the README file. Please check.

**Comment by isharailanga** [comment]
> > > Please do the changes required for the readme file and @isharailanga please review that
> > 
> > 
> > Hi @DivyaPremanantha @DoryFazna,
> > Ack. Please keep me posted.
> 
> Updated the README file. Please review.

Line 26 in Readme:
Add file and use specification instead of Spec as follows:
Update the `<puppet_environment>/modules/ob_common/manifests/params.pp` **file** of **Puppetmaster** according to the Open Banking **specification** (UK, AU, Berlin). The default value is set as UK.

File: README.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ Each Puppet module contains the following ```.pp``` files.
 
 ## Customizations
 
-1.  When updating the `<puppet_environment>/modules/ob_common/manifests/params.pp` of **Puppetmaster** to point the created databases, 
+1.  When updating the `<puppet_environment>/modules/ob_common/manifests/params.pp` file of **Puppetmaster** to point the created databases,
```
Comment (isharailanga) [inline README.md:100]: We can get rid of the numbering for this section

Comment (DoryFazna) [reply README.md]: Done

**Review by isharailanga — APPROVED** [review]
Approving the README.md file.

**Comment by Dinushi** [comment]
There is few more updates needs to be done to the following template file.

https://raw.githubusercontent.com/wso2/puppet-open-banking/master/modules/obam/templates/carbon-home/repository/resources/api_templates/velocity_template.xml.erb 

There are few new changes in WUM related to "cds-admin" in the velocity_template.xml. Please add those changes as well.

**Review by Dinushi — CHANGES_REQUESTED** [review]
There is few more updates needs to be done to the following template file.

https://raw.githubusercontent.com/wso2/puppet-open-banking/master/modules/obam/templates/carbon-home/repository/resources/api_templates/velocity_template.xml.erb

There are few new changes in WUM related to "cds-admin" in the velocity_template.xml. Please add those changes as well.

**Review by VimukthiPerera — APPROVED** [review]

**Comment by DoryFazna** [comment]
> There is few more updates needs to be done to the following template file.
> 
> https://raw.githubusercontent.com/wso2/puppet-open-banking/master/modules/obam/templates/carbon-home/repository/resources/api_templates/velocity_template.xml.erb
> 
> There are few new changes in WUM related to "cds-admin" in the velocity_template.xml. Please add those changes as well.

Updated. Thanks.

**Review by Dinushi — APPROVED** [review]

**Review by DivyaPremanantha — APPROVED** [review]

**Pull request merged**